.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -95,7 +95,6 @@ jobs:
   test-lint-rules:
     name: Test (Lint Rules)
     runs-on: ubuntu-latest
-    timeout-minutes: 10
```

Comment:
It's never us. It's [always](https://github.com/CrowdStrike/glide-core/actions/runs/16881760713/job/47819251239) Ubuntu's Azure instance.